analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/CalculationsShouldNotOverflow.CSharp11.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Sample
+    {
+        public void ListPattern(int[] array)
+        {
+            if (array is [2147483600, ..])
+                _ = array[0] + 100; // FIXME Non-compliant
```

Comment:
Why is it FIXME? I doubt this was supported by the old engine

Reply:
It was not. I just thought this was something we wanted to fix in this sprint. Do we only use FIXME if it would be a regression otherwise?

Reply:
During migration, we use FIXME to change `Noncompliant` to `FIXME Non-compliant` so we have a reference of what was supported before and what wasn't.